src/processing/vFramer/src/other_drawers.cpp

```diff
@@ -268,3 +268,99 @@ void overlayStereoDraw::draw(cv::Mat &image, const ev::vQueue &eSet, int vTime)
         }
     }
 }
+
+//Rasterplot Draw//
+//=============//
+
+const std::string rasterDraw::drawtype = "RASTER";
+
+std::string rasterDraw::getDrawType()
+{
+    return rasterDraw::drawtype;
+}
+
+std::string rasterDraw::getEventType()
+{
+    return AddressEvent::tag;
+}
+
+void rasterDraw::draw(cv::Mat &image, const ev::vQueue &eSet, int vTime)
+{
+    // eSet is a merge of all queues received by the vFramer:i-port in frameRate time
+    // The vFramer:i-port accumulates events, so the eSet gets longer over time!
+
+    // check if eSet contains elements
+    if(eSet.empty()){
+        return;
+    }
+    // get latest timestamp
+    if(vTime < 0){
+        vTime = eSet.back()->stamp;
+    }
+    // Reverse Timeelement-Iterator:
+    for(int y=(Ylimit-1.0); y>=0; y--){
```

Comment:
there should be no need for pixel storage at all. It should be a matter of taking all the events in eSet and plotting them on the image as you like. I don't think the implementation is one we want to use.